python/tests/integration/toolbox/test_library_tool.py

```diff
@@ -82,18 +76,18 @@ def test_write_keys(lib_type, request):
     assert len(new_keys) == len(all_keys)
 
 
-@pytest.mark.parametrize("lib_type", ["lmdb_version_store", "s3_version_store"])
-def test_count_keys(lib_type, request):
+def test_count_keys(object_and_lmdb_version_store):
     df = sample_dataframe()
```

Comment:
It will be easier to review if you do `lib = object_and_lmdb_version_store` here?

This is a common pattern used in a lot of tests as the fixture name can get rather long.

Note: I am reviewing from the last file up and there are many similar examples above.